src/azul/__init__.py

```diff
@@ -202,6 +202,10 @@ def sam_service_url(self):
     def dss_query_prefix(self) -> str:
         return os.environ.get('AZUL_DSS_QUERY_PREFIX', '')
 
+    @property
+    def partition_prefix_length(self) -> int:
+        return int(os.environ.get('AZUL_PARTITION_PREFIX_LENGTH', 2))
```

Comment:
Defaults go only into the top-level `environment.py` and shouldn't be duplicated here.